TemplateBundle/content/Application/.template.config/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "JaggerJo",
+  "classifications": [
+    "Console",
+    "Avalonia",
+    "UI",
+    "Elmish"
+  ],
+  "name": "Avalonia FuncUI Appication",
+  "tags": {
+    "language": "F#",
+    "type": "project"
+  },
+  "identity": "Avalonia.FuncUI.Application",
+  "shortName": "funcUI.application",
```

Comment:
maybe we could go with `QuickStart` or something as a name. I like that the other templates only have what is essentially required. Maybe **Full** is not the best name, but could be extended to have another simple page.

- **Basic** only includes one page
- **Full** only includes two pages with styles
- **QuckStart** includes everything you need to get started quickly

Reply:
I like the QuickStart name yeah I like it
I think I can look in the full after these changes

Reply:
nice!